<ui version="4.0" >
 <author>Tomeu Borras</author>
 <class>IncidenciaComercialBase</class>
 <widget class="QWidget" name="IncidenciaComercialBase" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>613</width>
    <height>414</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Ruta comercial</string>
  </property>
  <property name="windowIcon" >
   <iconset resource="../../../bulmages.qrc" >
    <normaloff>:/BfBulmaFact48x48/images/png/48x48/i_bulmages_ventas_facturas.48x48.png</normaloff>:/BfBulmaFact48x48/images/png/48x48/i_bulmages_ventas_facturas.48x48.png</iconset>
  </property>
  <layout class="QVBoxLayout" >
   <property name="spacing" >
    <number>5</number>
   </property>
   <property name="margin" >
    <number>5</number>
   </property>
   <item>
    <layout class="QHBoxLayout" >
     <property name="spacing" >
      <number>6</number>
     </property>
     <property name="margin" >
      <number>0</number>
     </property>
     <item>
      <layout class="QHBoxLayout" >
       <property name="spacing" >
        <number>6</number>
       </property>
       <property name="margin" >
        <number>0</number>
       </property>
       <item>
        <widget class="QLabel" name="textLabel1_2_2_3_8" >
         <property name="sizePolicy" >
          <sizepolicy vsizetype="Preferred" hsizetype="Minimum" >
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text" >
          <string>Fecha de la incidencia:</string>
         </property>
         <property name="buddy" >
          <cstring>mui_fechaincidenciacomercial</cstring>
         </property>
        </widget>
       </item>
       <item>
        <widget class="BlDateSearch" name="mui_fechaincidenciacomercial" />
       </item>
      </layout>
     </item>
     <item>
      <spacer>
       <property name="orientation" >
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0" >
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <layout class="QHBoxLayout" >
       <property name="spacing" >
        <number>6</number>
       </property>
       <property name="margin" >
        <number>0</number>
       </property>
       <item>
        <widget class="QLabel" name="textLabel1_2_2_3_6" >
         <property name="sizePolicy" >
          <sizepolicy vsizetype="Preferred" hsizetype="Minimum" >
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text" >
          <string>Referencia</string>
         </property>
         <property name="buddy" >
          <cstring>mui_refincidenciacomercial</cstring>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="mui_refincidenciacomercial" />
       </item>
       <item>
        <widget class="QToolButton" name="mui_informereferencia_2" >
         <property name="minimumSize" >
          <size>
           <width>32</width>
           <height>32</height>
          </size>
         </property>
         <property name="toolTip" >
          <string>Informe por Referencia</string>
         </property>
         <property name="whatsThis" >
          <string>Informe por Referencia</string>
         </property>
         <property name="text" >
          <string/>
         </property>
         <property name="icon" >
          <iconset resource="../../../bulmages.qrc" >
           <normaloff>:/BfBulmaFact48x48/images/png/48x48/i_bulmages_informexreferencia.48x48.png</normaloff>:/BfBulmaFact48x48/images/png/48x48/i_bulmages_informexreferencia.48x48.png</iconset>
         </property>
         <property name="iconSize" >
          <size>
           <width>32</width>
           <height>32</height>
          </size>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" >
     <property name="spacing" >
      <number>6</number>
     </property>
     <property name="margin" >
      <number>0</number>
     </property>
     <item>
      <widget class="QLabel" name="textLabel1_2_2_3_9" >
       <property name="sizePolicy" >
        <sizepolicy vsizetype="Preferred" hsizetype="Minimum" >
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text" >
        <string>Hora:</string>
       </property>
       <property name="buddy" >
        <cstring>mui_refincidenciacomercial</cstring>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="mui_horaincidenciacomercial" />
     </item>
     <item>
      <spacer>
       <property name="orientation" >
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0" >
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" >
     <property name="spacing" >
      <number>6</number>
     </property>
     <property name="margin" >
      <number>0</number>
     </property>
     <item>
      <layout class="QHBoxLayout" >
       <property name="spacing" >
        <number>6</number>
       </property>
       <property name="margin" >
        <number>0</number>
       </property>
       <item>
        <widget class="QLabel" name="textLabel1_2_2_3_11" >
         <property name="sizePolicy" >
          <sizepolicy vsizetype="Preferred" hsizetype="Minimum" >
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text" >
          <string>Cliente:</string>
         </property>
         <property name="buddy" >
          <cstring>mui_idcliente</cstring>
         </property>
        </widget>
       </item>
       <item>
        <widget class="BlSearchWidget" name="mui_idcliente" />
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" >
       <property name="spacing" >
        <number>6</number>
       </property>
       <property name="margin" >
        <number>0</number>
       </property>
       <item>
        <widget class="QLabel" name="textLabel1_2_2_3_7" >
         <property name="sizePolicy" >
          <sizepolicy vsizetype="Preferred" hsizetype="Minimum" >
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text" >
          <string>Trabajador:</string>
         </property>
         <property name="buddy" >
          <cstring>mui_idtrabajador</cstring>
         </property>
        </widget>
       </item>
       <item>
        <widget class="BfBuscarTrabajador" name="mui_idtrabajador" />
       </item>
      </layout>
     </item>
     <item>
      <spacer>
       <property name="orientation" >
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0" >
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" >
     <property name="spacing" >
      <number>6</number>
     </property>
     <property name="margin" >
      <number>0</number>
     </property>
     <item>
      <layout class="QHBoxLayout" >
       <property name="spacing" >
        <number>6</number>
       </property>
       <property name="margin" >
        <number>0</number>
       </property>
       <item>
        <widget class="QLabel" name="textLabel1_2_2_3_12" >
         <property name="sizePolicy" >
          <sizepolicy vsizetype="Preferred" hsizetype="Minimum" >
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text" >
          <string>Estado de la incidencia:</string>
         </property>
         <property name="buddy" >
          <cstring>mui_estadoincidenciacomercial</cstring>
         </property>
        </widget>
       </item>
       <item>
        <widget class="BusquedaEstadoIncidencia" name="mui_estadoincidenciacomercial" />
       </item>
      </layout>
     </item>
     <item>
      <spacer>
       <property name="orientation" >
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0" >
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QLabel" name="textLabel1_2_2_3_5" >
     <property name="sizePolicy" >
      <sizepolicy vsizetype="Preferred" hsizetype="Minimum" >
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text" >
      <string>Comentarios:</string>
     </property>
     <property name="buddy" >
      <cstring>mui_comentincidenciacomercial</cstring>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QTextEdit" name="mui_comentincidenciacomercial" />
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="2" margin="2" />
 <customwidgets>
  <customwidget>
   <class>BusquedaEstadoIncidencia</class>
   <extends>QComboBox</extends>
   <header>busquedaestadoincidencia.h</header>
  </customwidget>
  <customwidget>
   <class>BfBuscarTrabajador</class>
   <extends>QComboBox</extends>
   <header>bfbuscartrabajador.h</header>
  </customwidget>
  <customwidget>
   <class>BlDateSearch</class>
   <extends>QLineEdit</extends>
   <header>bldatesearch.h</header>
  </customwidget>
  <customwidget>
   <class>Busqueda</class>
   <extends>QLineEdit</extends>
   <header>blsearchwidget.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../../../bulmages.qrc" />
 </resources>
 <connections/>
</ui>
